src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -160,6 +156,16 @@ message GetClusterResourceStateReply {
   repeated ClusterResourceConstraint cluster_resource_constraints = 6;
 }
 
+message PendingInstanceRequest {
+  // The instance type name for the cloud, e.g. t1.micro
+  string instance_type_name = 1;
+  // The node type name, e.g. ray-head-node, matching `available_node_types`
+  // in the autoscaler config. See `ray/autoscaler/ray-schema.json`
+  string ray_node_type_name = 2;
```

Comment:
can we include `string ray_node_type_name = 2;` in `NodeState`?

Reply:
+1. We can pass it in using the same way as `string instance_id = 12;` (i.e. passed in via a new environment variable)

Reply:
I guess that's possible.